.github/workflows/run.yml

```diff
@@ -0,0 +1,70 @@
+name: tox-ansible
```

Comment:
Are we sure that we want this in the workflows for this repository? This repo isn't likely to host Ansible code itself, is it? I realize the point is to have it shared across other users of the plugin if they don't have a reason to customize the matrix, but it seems unnecessary to have it run on this repo directly.

Reply:
It should be disabled now in the actions, yeah, the intention wasn't to have it run here, just hosted.

I'll talk to sorin about this, it might make sense to have them all in one palce outside of the py project repos.

Reply:
I think we can just punt on this for now........  We are talking about we can centalize these and where it's appropriate.  The other thing that comes into play is the idea of a "content developers kit" for ansible that acts as an installer for a number of python project scattered all over the place. 

So for now, I say we leave the workflow here until we solve the bigger structure question for all the dev tools.